tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/ExportCsvTest.java

```diff
@@ -33,5 +39,40 @@ public void exportCsv() throws IOException {
 
         File appTagsCsv = new File(tmp.getRoot(), "ApplicationFileTechnologies.csv");
         assertTrue(appTagsCsv.exists());
+
+        List<List<String>> records = readCSVFile(appTagsCsv);
+        records.forEach(record ->
```

Comment:
it's ok but as a suggestion only : 
assertTrue(records.allMatch(this::isRowSorted))

Reply:
I like this, suggested change has been  made